deploy/stos.subscription.yaml

```diff
@@ -7,4 +7,4 @@ spec:
   source: olm-operators
   sourceNamespace: olm
   name: storageos
-  channel: alpha
+  channel: stable
```

Comment:
This is used in the e2e test setup. And changing this without changing https://github.com/storageos/cluster-operator/blob/master/deploy/olm-operators.configmap.yaml#L471 will fail the operator installation. But I'm working on changing the configmap a bit, so let's remove this change and just change the channel in https://github.com/storageos/cluster-operator/blob/master/deploy/olm/storageos/storageos.package.yaml#L4 only.